botx/bots.py

```diff
@@ -648,6 +652,48 @@ def decorator(handler: typing.ExceptionHandler) -> Callable:
             payload=StealthDisablePayload(group_chat_id=chat_id,),
         )
 
+    async def add_users(
+        self,
+        credentials: sending.SendingCredentials,
+        chat_id: UUID,
+        users_huids: List[UUID],
+    ) -> None:
+        """Add users to chat
+
+        Arguments:
+            credentials: credentials of chat.
+            chat_id: id of chat to disable stealth,
```

Comment:
Похоже, что тут описание забыл поменять после копирования из другой фукнции
```suggestion
            chat_id: id of chat to add users,
```